spec/features/merchants/invoices/show_spec.rb

```diff
@@ -67,8 +59,8 @@
 
     visit "/merchants/#{merchant1.id}/invoices/#{invoice1.id}"
 
-    within("#invoice_#{item.id}") do
-      expect(page).to have_content("Status: shipped")
+    within("#status-#{item.id}") do
```

Comment:
perhaps change to status_ID rather than status-ID? super small thing, no big deal.